buster/docparser.py

```diff
@@ -66,7 +67,7 @@ def get_all_documents(
         urls.extend(urls_file)
         names.extend(names_file)
 
-    documents_df = pd.DataFrame.from_dict({"name": names, "url": urls, "text": sections})
```

Comment:
Note that this change of variables here will have an impact on many other regions in the code... perhaps it is best to revert to the original variable names and have a separate PR where all instances are changed.

Reply:
Fair enough, I renamed the columns when reading the documents, and added a TODO to fix it.

Reply:
I still see it being changed in line 70? (right above this comment)

Reply:
sorry, I meant I did a df.rename() when read/writing, with a TODO to change this in a futur PR